hspec-discover/test/Test/Hspec/Discover/RunSpec.hs

```diff
@@ -38,7 +41,11 @@ spec = do
         , "main = hspec spec"
         , "spec :: Spec"
         , "spec = " ++ unwords [
-               "postProcessSpec \"test-data/nested-spec/Foo/Bar/BazSpec.hs\" (describe \"Foo.Bar.Baz\" Foo.Bar.BazSpec.spec)"
+               "postProcessSpec \"test-data/nested-spec/Dir8/X64Spec.hs\" (describe \"Dir8.X64\" Dir8.X64Spec.spec)"
+          , ">> postProcessSpec \"test-data/nested-spec/Dir8/X128Spec.hs\" (describe \"Dir8.X128\" Dir8.X128Spec.spec)"
+          , ">> postProcessSpec \"test-data/nested-spec/Dir16/X48Spec.hs\" (describe \"Dir16.X48\" Dir16.X48Spec.spec)"
+          , ">> postProcessSpec \"test-data/nested-spec/Dir16/X160Spec.hs\" (describe \"Dir16.X160\" Dir16.X160Spec.spec)"
+          , ">> postProcessSpec \"test-data/nested-spec/Foo/Bar/BazSpec.hs\" (describe \"Foo.Bar.Baz\" Foo.Bar.BazSpec.spec)"
```

Comment:
I think there is no really need to test this here.  Unit tests for the sort function are good enough.